cypress-public.config.ts

```diff
@@ -18,14 +17,17 @@ export default defineConfig({
           return null;
         },
         setAllowedTerminalIpAddresses(ipAddresses) {
-          return setAllowedTerminalIpAddresses(ipAddresses);
+          return toggleFeatureFlagFromPostgres({
```

Comment:
It sounds like you maybe already are planning on doing this, but just a recommendation to drop "FromPostgres" etc. to remain implementation-agnostic.